src/runtime/parallel/slave.js

```diff
@@ -26,12 +28,26 @@ export default class Slave {
     this.cwd = cwd
     this.eventBroadcaster = new EventEmitter()
     this.stackTraceFilter = new StackTraceFilter()
+
+    const pathSepRegexp = new RegExp(escapeStringRegexp(path.sep), 'g')
+    const pathToRemove =
+      cwd.replace(pathSepRegexp, path.posix.sep) + path.posix.sep
+    this.pathRegexp = new RegExp(escapeStringRegexp(pathToRemove), 'g')
```

Comment:
This path cleanup should go in the formatters and should be its own PR. I'd prefer we replace the cwd with `.` instead of removing it entirely